docs/Makefile

```diff
@@ -16,7 +16,7 @@ clean:
 	rm -rf $(BUILDDIR)/*
 
 deps:
-	pip install --disable-pip-version-check -r requirements.txt
```

Comment:
Why was this removed? Without that flag you get the annoying upgrade message all the time.